tests/lib/linter/linter.js

```diff
@@ -9256,6 +9258,147 @@ describe("Linter with FlatConfigArray", () => {
+            assert.strictEqual(messages.length, 1);
+            assert.deepStrictEqual(messages[0], {
+                ruleId: null,
+                severity: 1,
+                message: "No matching configuration found for C:\\file.js.",
+                line: 0,
+                column: 0,
+                nodeType: null
+            });
+
+            messages = linterWithOptions.verify(code, config, "C:\\file.ts");
+            assert.strictEqual(messages.length, 1);
+            assert.deepStrictEqual(messages[0], {
+                ruleId: null,
+                severity: 1,
+                message: "No matching configuration found for C:\\file.ts.",
+                line: 0,
+                column: 0,
+                nodeType: null
+            });
+
+            messages = linterWithOptions.verify(code, config, "D:\\foo\\file.js");
+            assert.strictEqual(messages.length, 1);
+            assert.deepStrictEqual(messages[0], {
+                ruleId: null,
+                severity: 1,
+                message: "No matching configuration found for D:\\foo\\file.js.",
+                line: 0,
+                column: 0,
+                nodeType: null
+            });
+
+            messages = linterWithOptions.verify(code, config, "D:\\foo\\file.ts");
+            assert.strictEqual(messages.length, 1);
+            assert.deepStrictEqual(messages[0], {
+                ruleId: null,
+                severity: 1,
+                message: "No matching configuration found for D:\\foo\\file.ts.",
+                line: 0,
+                column: 0,
+                nodeType: null
+            });
+
+            messages = linterWithOptions.verify(code, config, "C:\\foo\\file.js");
+            assert.strictEqual(messages.length, 1);
+            assert.strictEqual(messages[0].ruleId, "test/test-rule-1");
+
+            messages = linterWithOptions.verify(code, config, "C:\\foo\\file.ts");
+            assert.strictEqual(messages.length, 2);
+            assert.strictEqual(messages[0].ruleId, "test/test-rule-1");
+            assert.strictEqual(messages[1].ruleId, "test/test-rule-2");
+
+            messages = linterWithOptions.verify(code, config, "C:\\foo\\bar\\file.ts");
+            assert.strictEqual(messages.length, 3);
+            assert.strictEqual(messages[0].ruleId, "test/test-rule-1");
+            assert.strictEqual(messages[1].ruleId, "test/test-rule-2");
+            assert.strictEqual(messages[2].ruleId, "test/test-rule-3");
+        });
+
+        it("should ignore external files with Posix paths", () => {
+            const configs = createFlatConfigArray(
+                { files: ["**/*.js"] },
+                { basePath: "/foo" }
+            );
+
+            configs.normalizeSync();
+            const messages1 = linter.verify("foo", configs, "/foo/bar.js", true);
+            const messages2 = linter.verify("foo", configs, "/bar.js", true);
```

Comment:
```suggestion
            const messages1 = linter.verify("foo", configs, "/foo/bar.js");
            const messages2 = linter.verify("foo", configs, "/bar.js");
```

`Linter#verify()` accepted only 3 arguments.

Reply:
Good catch! I fixed all usages in 4eb8d116895104b6c931c29328c60a3e58b06fc2.